kok-core/src/main/java/com/kok/kokcore/room/domain/Room.java

```diff
@@ -31,6 +31,15 @@ public Room(String id, String roomName, int capacity, String hostProfile,
     }
 
     public static Room create(String roomName, int capacity, String hostProfile, String password) {
+        validateParameter(roomName, capacity, hostProfile);
+
+        String roomId = UUID.randomUUID().toString();
+        String roomLinkUrl = ROOM_LINK_URL + roomId;
+
+        return new Room(roomId, roomName, capacity, hostProfile, password, roomLinkUrl);
+    }
+
+    private static void validateParameter(String roomName, int capacity, String hostProfile) {
         if (roomName == null || roomName.trim().isEmpty()) {
```

Comment:
```suggestion
        if (roomName == null || roomName.trim().isBlank()) {
```
공백 문자열도 허용하지 않을 것 같아서요 ㅎㅎ